lua/apisix/plugins/proxy-rewrite.lua

```diff
@@ -49,6 +49,11 @@ local schema = {
             type = "object",
             minProperties = 1,
         },
+        resp_headers = {
+            description = "new headers for repsonse",
+            type = "object",
+            minProperties = 1,
```

Comment:
Check keys and values are strings, and other types are not allowed, such as: table.